### PR TITLE
Update rules for OCS adding actions "Child entities" & "groups_id_tech"

### PR DESCRIPTION
**Pull request by keha35**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #4484 

Update rules for "assigning an item to an entity" adding actions "Child entities" & "groups_id_tech"

### CODE REVIEW

File: inc/ruleimportentity.class.php [hunk 1]
```diff
@@ -168,6 +168,16 @@ function getActions() {
 
       $actions['_ignore_import']['name']   = __('To be unaware of import');
       $actions['_ignore_import']['type']   = 'yesonly';
+      $actions['_ignore_import']['type']   = 'yesonly';
```
Comment (cedric-anne) [inline inc/ruleimportentity.class.php]: Line is duplicated.

File: inc/ruleimportentity.class.php [hunk 2]
```diff
@@ -169,6 +169,15 @@ function getActions() {
       $actions['_ignore_import']['name']   = __('To be unaware of import');
       $actions['_ignore_import']['type']   = 'yesonly';
 
+      $actions['is_recursive']['name']     = __('Child entities');
+      $actions['is_recursive']['type']     = 'yesno';
+      $actions['is_recursive']['table']    = '';
```
Comment (orthagh) [inline inc/ruleimportentity.class.php]: if table empty, don't add the line

Comment (orthagh) [reply inc/ruleimportentity.class.php]: this one:
```
$actions['is_recursive']['table']    = '';
```

**Comment by orthagh** [comment]
Thank you

**Comment by orthagh** [comment]
You pushed on master for this one, do you need it on 9.3/bugfixes ?

**Comment by keha35** [comment]
Yes if it's possible it would be great.

**Comment by orthagh** [comment]
I'll do it for this PR, but for future ones, target 9.3/bugfixes as base branche for your PRs.

**Comment by keha35** [comment]
Ok, thanks.

**Pull request merged**